eras/alonzo/impl/src/Cardano/Ledger/Alonzo/Tools.hs

```diff
@@ -47,7 +56,7 @@ data ScriptFailure c
   | -- | Missing datum.
     MissingDatum (DataHash c)
   | -- | Plutus evaluation error.
-    ValidationFailed P.EvaluationError
+    ValidationFailed PV1.EvaluationError [Text]
```

Comment:
This is rather confusing - what happens to Plutus V2 evaluation errors? I guess the key is that presently these types are the same, but it would be good to clarify this at least in the documentation.

Reply:
If we're going to make it very mechanical, then I think we should have different errors here. Or an error GADT indexed by the language enum? :trollface:

Reply:
good catch!